content/blog/10-pearls-with-azure-functions-in-pulumi/index.md

```diff
@@ -0,0 +1,361 @@
+---
+title: "10 Pearls With Azure Functions in Pulumi"
+date: 2019-08-15
+draft: true
+meta_image: meta.png
+authors: ["mikhail-shilkov"]
+tags: ["Azure", "Serverless"]
+meta_desc: "In this post, we'll take a look at 10 pearls&mdash;bite-sized code snippets&mdash;that demonstrate using Pulumi to build serverless applications with Azure Functions and infrastructure as code."
+---
+
+In this post, we'll take a look at 10 "pearls"&mdash;bite-sized code snippets&mdash;that demonstrate using Pulumi to build serverless applications with Azure Functions and infrastructure as code. These pearls are organized into four categories, each demonstrating a unique scenario:
+
+- **Function App Deployment**: Take an existing Azure Functions application in your language of choice and deploy it with infrastructure as code.
```

Comment:
```suggestion
- **Function App Deployment**: Deploy an Azure Functions application using infrastructure as code.
```

Reply:
Don't you think this becomes too broad - the other three bullet points fit this shorter definition too.